DisCatSharp/Net/Rest/DiscordApiClient.cs

```diff
@@ -2361,9 +2361,10 @@ internal async Task<DiscordChannel> CreateGuildChannelAsync(ulong guildId, strin
 	/// <param name="postCreateUserRateLimit">The per user post create rate limit.</param>
 	/// <param name="defaultAutoArchiveDuration">The default auto archive duration.</param>
 	/// <param name="defaultSortOrder">The default sort order.</param>
+	/// <param name="defaultLayout">The default layout.</param>
 	/// <param name="flags">The channel flags.</param>
 	/// <param name="reason">The reason.</param>
-	internal async Task<DiscordChannel> CreateForumChannelAsync(
+	internal async Task<DiscordChannel> CreateGuildForumChannelAsync(
```

Comment:
I'm unsure why this method name would be changed, this is a breaking change.
Without it still is, but affects less people.

I don't know if this is a new change or whether i missed it last time. I apologize if it's the latter.

Reply:
I approved it because it's an internal change not facing to devs ever